packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -191,14 +194,28 @@ describe('Base Deploy Retrieve Commands', () => {
     it('should use the api version from SFDX configuration', async () => {
       const executor = new TestDeployRetrieve();
       const configApiVersion = '30.0';
-      sb.stub(ConfigUtil, 'getConfigValue')
-        .withArgs('apiVersion')
-        .returns(configApiVersion);
+      sb.stub(ConfigUtil, 'getUserConfiguredApiVersion').resolves(
+        configApiVersion
+      );
+      const getOrgApiVersionSpy = sb.spy(OrgAuthInfo, 'getOrgApiVersion');
 
       await executor.run({ data: {}, type: 'CONTINUE' });
       const components = executor.lifecycle.doOperationStub.firstCall.args[0];
 
       expect(components.apiVersion).to.equal(configApiVersion);
+      expect(getOrgApiVersionSpy.called).to.be.false;
```

Comment:
nit: `expect(getOrgApiVersionSpy.called).to.equal(false);` does the same assertion without the linter being overriden or beefing.